notebooks/shield-dash-import.py

```diff
@@ -22,8 +24,12 @@ def getShieldProps(p):
     }
     for k in ['firstrun', 'msg', 'name', 'variation']:
         if k == 'firstrun':
-            out[k] = datetime.datetime.utcfromtimestamp(
-                int(p['payload'][k]) // 1e3)
+            firstrun = p['payload'].get(k)
+            if firstrun is None:
+                out[k] = out['creation_date']
```

Comment:
Was this change necessary? I'm not sure we want to conflate creation date with firstrun, which has a pretty specific use for these studies and isn't the same as creation_date.